app/hooks/use-global-polling.ts

```diff
@@ -0,0 +1,56 @@
+import { useCallback, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { DEFAULT_POLLING_INTERVAL } from '@constants/index';
+import { selectAddress } from '@store/keys';
+import {
+  fetchBlockTimeInfo,
+  fetchCoreDetails,
+  fetchStackerInfo,
+  fetchStackingInfo,
+} from '@store/stacking';
+import { getAddressTransactions } from '@store/transaction';
+import { getAddressDetails } from '@store/address';
+import { useNavigatorOnline } from './use-navigator-online';
+import { useInterval } from './use-interval';
+import { useApi } from './use-api';
```

Comment:
Is there any specific convention we follow for how we order our imports?

Reply:
Not exactly. I tend to follow: React > Other third party libs > Internal aliased imports > relative imports.

I've tried tirelessly to get an eslint import order set up going, but https://github.com/benmosher/eslint-plugin-import misses some key rules to get it right (i.e. putting react first).